server/sonar-web/src/main/js/apps/permission-templates/views/GroupsView.js

```diff
@@ -23,7 +23,8 @@ import '../../../components/SelectList';
 
 function getSearchUrl(permission, permissionTemplate) {
   return window.baseUrl +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3512)